packages/flutter_test/lib/src/_binding_io.dart

```diff
@@ -73,8 +73,21 @@ void mockFlutterAssets() {
 /// If another [HttpClient] is provided using [HttpOverrides.runZoned], that will
 /// take precedence over this provider.
 class _MockHttpOverrides extends HttpOverrides {
+  bool warningPrinted = false;
   @override
   HttpClient createHttpClient(SecurityContext _) {
+    if (!warningPrinted) {
```

Comment:
This would output the message for any widget test which incidentally built a network image. It doesn't seem desirable if their test was otherwise working.

Reply:
Is there some way to just do this on a test failure?

Reply:
I could try to use test_package.printOnFailure, but I'm not sure how to test that anymore - hah

Reply:
I guess I can put it in a new test file and run it under a harness that asserts the test fails and greps the message.

Reply:
Ok - now it should only print the warning if a test fails and the HttpClient was created.